controllers/rest/api.go

```diff
@@ -369,14 +370,22 @@ func (p *RESTAPI) GetWeOSConfig() *APIConfig {
 	return p.Config
 }
 
-//RegisterLog setup a log
+// RegisterLog setup a log
 func (p *RESTAPI) RegisterLog(name string, logger model.Log) {
 	if p.logs == nil {
 		p.logs = make(map[string]model.Log)
 	}
 	p.logs[name] = logger
 }
 
+func (p *RESTAPI) RegisterLogger(logger model.Log) {
+	p.logger = logger
+}
+
+func (p *RESTAPI) GetLogger() model.Log {
```

Comment:
why not replace GetLog so that way the change will propogate across all our microservices